manifests/o/OpenWhisperSystems/Signal/5.29.0/OpenWhisperSystems.Signal.locale.en-US.yaml

```
@@ -0,0 +1,37 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: OpenWhisperSystems.Signal
+PackageVersion: 5.29.0
+PackageLocale: en-US
+Publisher: Open Whisper Systems
+PublisherUrl: https://www.signal.org
+PublisherSupportUrl: https://github.com/signalapp/Signal-Desktop/issues
+PrivacyUrl: https://www.signal.org/legal/#privacy-policy
+Author: Open Whisper Systems
+PackageName: Signal
+PackageUrl: https://www.signal.org
+License: GPLv3
+LicenseUrl: https://www.gnu.org/licenses/gpl-3.0.en.html
+Copyright: Copyright 2014-2021 Open Whisper Systems
+CopyrightUrl: https://www.signal.org/legal/#terms-of-service
+ShortDescription: Signal is an encrypted communications application.
+Description: Signal is an encrypted communications application. It uses the Internet to send one-to-one and group messages, which can include files, voice notes, images and videos, and make one-to-one voice and video calls.
+Moniker: signal
+Tags:
+- chat
+- cross-platform
+- encryption
+- foss
+- messaging
+- open-source
+- privacy
+- security
+- texting
+# Agreements: 
+ReleaseNotes: |
+  - v5.29.0 Now, with fewer frantic searches for the loud typist or heavy chewer! In the call lobby, Desktop will now automatically disable audio if there are eight or more people in the call.
+  - v5.28.0 New year, new initial setup screen. We've redesigned the initial setup screens just in time for 2022.
+ReleaseNotesUrl: https://github.com/signalapp/Signal-Desktop/releases
+ManifestType: defaultLocale
```

If possible the ReleaseNotesUrl should point to the specific version, definitively when its GitHub as the release page can get MESSY
I don't think the 5.28 patch note is necessary?
```suggestion
ReleaseNotes: |
  • Now, with fewer frantic searches for the loud typist or heavy chewer! In the call lobby, Desktop will now automatically disable audio if there are eight or more people in the call.
ReleaseNotesUrl: https://github.com/signalapp/Signal-Desktop/releases/tag/v5.29.0
ManifestType: defaultLocale
```
edit before commenting:
I guess you have the 5.28 patch not there because we don't have the 5.28 version, feel free to add 5.28 just so we have it

---

will do